src/components/Person/Person.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${name}`}</h2>
```

Comment:
```suggestion
    <h2 className="Person__name">
      My name is {name}
    </h2>
```
You can just use JSX syntax without interpolation